includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -220,6 +224,82 @@ public function getData() {
 		return $data;
 	}
 
+	private function getWikiaOrgModel() {
+		$data = [
+			'header' => [
+				'type' => 'link-image',
+				// 'image' is deprecated, use 'image-data' instead
```

Comment:
What's this comment about. Can we just get rid of image then, or do we need to keep it for some reason?

Reply:
That's copied code and yes, definitely we can remove it,